### PR TITLE
update navbar link in sign Page

### PR DESCRIPTION
**Pull request by Rockyraven**
update navbar link in sign Page

### CODE REVIEW

File: Quickstore/pages/signin.html [hunk 1]
```diff
@@ -12,25 +12,34 @@
...
+                <div class="search-button">
+                    <a href="#"><i class="fa fa-search"></i></a>
+                </div>
+            </div>
+            <div class="login-section">
+                    <a href="/Quickstore/pages/signin.html"><i class="badge-icon fa fa-user"></i><p>Login</p></a>
+                    <div class="wishlist-badge badge">
+                        <a href="/Quickstore/pages/wishlist.html"> <i class="badge-icon fa fa-heart"></i> 
+                            <p class="names">wishlist</p>
+                            <span class="badge-symbol">3</span></a>
```
Comment (iamalhera) [inline Quickstore/pages/signin.html:32]: suggestion : badge-symbol makes less sense to the code, instead you can use badge-content

File: Quickstore/pages/signin.html [hunk 2]
```diff
@@ -12,25 +12,34 @@
...
+                    </div>
+                    <div class="cart-badge badge">
+                        <a href="/Quickstore/pages/cart.html"> <i class="badge-icon fa fa-shopping-cart"></i> 
+                            <p class="names">Cart</p>
```
Comment (iamalhera) [inline Quickstore/pages/signin.html:36]: here names class doesn't sound meaningful, try using meaningful class names

File: Quickstore/pages/signin.html (hunk 1 above)
Comment (Rockyraven) [reply Quickstore/pages/signin.html:32]: if i change it will become complicated i have also change in component libaray

**Comment by iamalhera** [comment]
Appreciation : UI is clean, loved your craft

File: Quickstore/pages/signin.html (hunk 2 above)
Comment (Rockyraven) [reply Quickstore/pages/signin.html:36]: i have changed class name

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to me. you can merge it

**Pull request merged**